lib/js/components/Badges/BadgeScore/BadgeScore.vue

```diff
@@ -7,11 +7,18 @@
 			'-fail': color === BADGE_SCORE_COLORS.FAIL,
 			'-inverted': color === BADGE_SCORE_COLORS.INVERTED,
 			'-neutral': color === BADGE_SCORE_COLORS.NEUTRAL,
+			'-neutral-medium': color === BADGE_SCORE_COLORS.NEUTRAL_MEDIUM,
 			'-small': size === BADGE_SCORE_SIZES.SMALL,
 			'-xsmall': size === BADGE_SCORE_SIZES.XSMALL,
 		}"
 	>
-		{{ text }}<span class="badgeScore__additionalText">{{ additionalText }}</span>
+		<div class="badgeScore__wrapper">
+			<wnl-icon v-if="icon" class="badgeScore__icon" :icon="icon" :size="iconSize" />
+			<div class="badgeScore__content">
```

Comment:
instead of separate `div` you could apply `align-self: baseline` on text and suffix

Reply:
We have there `align-items: baseline;` on parent, so true, it's redundant.

Reply:
I still see this container in HTML